### PR TITLE
documentation: ZENKO-1751_clean_up_SMB/CIFS_dox

### PR DESCRIPTION
**Pull request by wabernat**
This PR fixes, harmonizes, and reorders the configuration instructions for
CIFS/SMB and NFS features. You need it because it clarifies the instructions
and puts them in context (which are goods without qualification).

This PR fixes ZENKO-1751

### CODE REVIEW

**Comment by wabernat** [comment]
@bert-e approve

File: docs/docsource/Installation/configure/adding_CIFS_storage.rst [hunk 1]
```diff
@@ -35,26 +38,36 @@ Copy the following commands and run them with root or sudo privileges.
...
-3. Create a user for Cosmos.
+      In the CIFS/SMB use case, the NFS protocol and NFS version are ignored
+      and all configuration is through the config file below.
 
-4. Use information from the previous steps to configure instance-specific 
-   Cosmos values. Export the following variables with appropriate values
-   entered:
+#. Copy and save the location name.
+
+#. Create a bucket in this location.
```
Comment (ssalaues) [inline docs/docsource/Installation/configure/adding_CIFS_storage.rst]: Should probably specify the `Mirroring Mode` bucket creation. It's not a really a huge issue now but in the future when we support writing files this distinction will become crucial.

**Review by ssalaues — CHANGES_REQUESTED** [review]
just a small detail otherwise lgtm

File: docs/docsource/Installation/configure/adding_CIFS_storage.rst (hunk 1 above)
Comment (wabernat) [reply docs/docsource/Installation/configure/adding_CIFS_storage.rst]: I don't have a working NFS instance to test against, so I don't understand what this means. 

Do you have an NFS server I can "walk the steps" with? If not, can you take a screengrab of this step?

**Review by ssalaues — APPROVED** [review]

**Review by dora-korpar — APPROVED** [review]

**Review by tmacro — APPROVED** [review]

**Pull request merged**